cypress/src/assertions/index.js

```diff
@@ -359,3 +358,64 @@ export const assertWishlistCountWithWait = (count) => {
   cy.waitForWishlistPageLoaded();
   assertWishlistCount(count);
 };
+
+export const assertOrderImageDisplay = () => {
+  cy.get(".dropin-image--loaded")
+    .should('be.visible')
+}
+
+
+export const assertImageListDisplay = (selector, limit = null) => {
+  let imageQuery = cy.get(selector).find('img');
+
+  if (limit !== null) {
+    imageQuery = imageQuery.filter(`:lt(${limit})`);
+  }
+
+  imageQuery.each(($img) => {
+    cy.wrap($img)
+      .should('be.visible')
+      .and(($el) => {
+        expect($el[0].naturalWidth).to.be.greaterThan(0);
+      });
+  });
+};
+
+
+export const aseertSearchResults = () => {
```

Comment:
should be spelled `assertSearchResults`
```suggestion
export const assertSearchResults = () => {
```